lighthouse-core/report/html/renderer/element-screenshot-renderer.js

```diff
@@ -172,14 +178,13 @@ class ElementScreenshotRenderer {
         elementRectSC,
         maxLightboxSize
       );
-      if (!screenshotElement) return;
 
+      // This would be unexpected here. The null return is used to not create a thumbnail in details-renderer
```

Comment:
I'm not sure I understand the null *return* bit. is this saying `screenshotElement` is null only when there isn't a thumbnail so they shouldn't have been able to click on anything to get here?

Reply:
yes that's accurate.

Reply:
```suggestion
      // This would be unexpected here. 
      // When `screenshotElement` is `null`, there is also no thumbnail element for the user to have clicked to make it this far. 
```

how do you feel about this? :D